src/Distributions.jl

```diff
@@ -21,6 +21,13 @@ if isdefined(Base, :scale)
     import Base: scale
 end
 
+if isdefined(Base, :MathConstants)
+    import Base.MathConstants: γ
+else
+    # TODO replace with Compat
+    const ℯ = e
```

Comment:
I'm collecting things that should go in `Compat`. Meanwhile, this package uses `e` enough times that I think it's worth putting a temperary solution in place.

There's one occurance in the test that I didn't replace since it won't affect package users.

Reply:
When Base.MathConstants is defined, ASCII `e` is defined therein, so you could do the same thing I did a couple lines up with `γ`. That is,
```julia
if isdefined(Base, :MathConstants)
    import Base.MathConstants: γ, e
end
```
When MathConstants isn't defined, `e` will be defined in Main since it's exported from Base.

Reply:
It's done this way to match the new default behavior.